Tests/Fixtures/App/Kernel.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+namespace Doctrine\Bundle\PHPCRBundle\Tests\Fixtures\App;
+
 use Symfony\Cmf\Component\Testing\HttpKernel\TestKernel;
 use Symfony\Component\Config\Loader\LoaderInterface;
 
-class AppKernel extends TestKernel
+class Kernel extends TestKernel
```

Comment:
@xabbuh this i the kernel for this specific test case. It lives mostly in testing component.